docs/OPENSHIFT_GUIDE.md

```diff
@@ -0,0 +1,83 @@
+# KFServing on OpenShift
+
+To install standalone KFServing on OpenShift, the easiest way is to use the [`quick_install.sh`](./hack/quick_install.sh) script provided
+in this repository. This assumes you do not already have Istio and Knative running on your cluster.
+
+**Note**: These instructions were tested on OpenShift 4.5.15.
```

Comment:
```suggestion
**Note**: These instructions were tested on OpenShift 4.5.15, with Istio 1.3.1 and KNative 0.14.3. Our recommendation is to move to Istio 1.6+ and KNative 0.15+
```

Reply:
Wouldn't be accurate to say these versions since I tested standalone installation with whatever is in the quick_install script (i.e. Istio 1.6.2 and Knative 0.15.0)

Reply:
how about the changed line

Reply:
Updated with both suggestions.